### PR TITLE
changes to add help improve and child benefit banner

### PR DESCRIPTION
**Pull request by kritikarai6**

### CODE REVIEW

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -54,7 +54,10 @@ sca-wrapper {
       pertax-frontend.url = "http://localhost:9232"
       accessibility-statement-frontend.url = "http://localhost:12346"
       single-customer-account-wrapper-data.url = "http://localhost:8422"
+      help-improve-banner.url = "https://signup.take-part-in-research.service.gov.uk/home?utm_campaign=PTAhomepage&utm_source=Other&utm_medium=other&t=HMRC&id=209"
```
Comment (yangwong9664) [inline src/main/resources/reference.conf:57]: utm_campaign should probably be a parameter that a service can set in their application.conf file

File: src/main/scala/uk/gov/hmrc/sca/config/AppConfig.scala [hunk 2]
```diff
@@ -70,10 +70,14 @@ class AppConfig @Inject()(configuration: Configuration, messages: MessagesApi) {
   val contactFrontendUrl: String = s"${configuration.get[String]("sca-wrapper.services.contact-frontend.url")}/contact/beta-feedback"
   val accessibilityStatementUrl: String = configuration.get[String]("sca-wrapper.services.accessibility-statement-frontend.url")
   val scaWrapperDataUrl = s"${configuration.get[String]("sca-wrapper.services.single-customer-account-wrapper-data.url")}/single-customer-account-wrapper-data"
-
+  val helpImproveBannerUrl: Option[String] = configuration.get[Option[String]]("sca-wrapper.services.help-improve-banner.url")
+  lazy val childBenefitLinkUrl = Some(
+    "https://docs.google.com/forms/d/e/1FAIpQLSegbiz4ClGW0XkC1pY3B02ltiY1V79V7ha0jZinECIz_FvSyg/viewform"
```
Comment (yangwong9664) [inline src/main/scala/uk/gov/hmrc/sca/config/AppConfig.scala:75]: can this url be moved to .conf?

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

**Review by kanchan555-ui — DISMISSED** [review]
approved

File: src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala [hunk 3]
```diff
@@ -65,10 +65,11 @@ class WrapperService @Inject()(
              backLinkUrl: String = "#",
              showSignOutInHeader: Boolean = false,
              scripts: Option[Html] = None,
-             showChildBenefitBanner: Boolean = false,
+             showChildBenefitBanner: Boolean = appConfig.showChildBenefitBanner,
              showUserResearchBanner: Boolean = false,
```
Comment (yangwong9664) [inline src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala:69]: can showUserResearchBanner be removed, and showHelpImproveBanner moved in its place instead? I think the user research banner was going to be the helpimprove banner but now it's not needed

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

**Review by yangwong9664 — APPROVED** [review]

**Pull request merged**